### PR TITLE
Return deliverables not including submissions as part of module store

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#1966** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1966?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1965** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1965?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1964** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1964?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1963** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1963?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1962** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1962?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDeliverableModel.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    val moduleId: ModuleId,
+    val name: String,
+    val position: Int,
+    val required: Boolean,
+    val sensitive: Boolean,
+    val type: DeliverableType,
+) {
+  companion object {
+    fun of(record: DeliverablesRecord): ModuleDeliverableModel =
+        with(DELIVERABLES) {
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDeliverableModel.kt:24]: Ah, using `with` on these `of(Record)` functions is a good idea.

File: src/test/kotlin/com/terraformation/backend/accelerator/db/ModuleStoreTest.kt [hunk 2]
```diff
@@ -601,4 +604,71 @@ class ModuleStoreTest : DatabaseTest(), RunsAsUser {
...
+          listOf(model2, model1),
+          store.fetchAllModules().firstOrNull()?.deliverables,
+          "Fetch all with deliverables")
+
+      insertCohortModule(cohortId, moduleId)
+
+      assertEquals(
+          listOf(model2, model1),
+          store.fetchModulesForProject(projectId).firstOrNull()?.deliverables,
+          "Fetch by project with deliverables")
```
Comment (sgrimm) [inline src/test/kotlin/com/terraformation/backend/accelerator/db/ModuleStoreTest.kt]: Maybe put the deliverables in separate modules and only put the project's cohort in one of the modules, to test that the function isn't returning all the deliverables with no filtering?

**Review by sgrimm — APPROVED** [review]
One minor test suggestion, but this looks good.

**Comment by tommylau523** [comment]
### Merge activity

* **May 20, 3:32 PM EDT**: @tommylau523 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1964).
* **May 20, 3:44 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1964) rebased this pull request as part of a merge.
* **May 20, 3:54 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1964).

**Pull request merged**